### PR TITLE
Feature/143 address field

### PR DESCRIPTION
**Pull request by pieroit**
#143 and #140

### CODE REVIEW

File: src/admin/WL_Metabox/WL_Metabox_Field_address.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	/**
+	 * Constructor.
+	 * 
+	 * @param array $args Set of fields containing info to build the subfields.
+	 */
+	public function __construct( $args ) {
+		
+		// leverage the WL_Metabox_Field class to build the subfields
+		$this->subfields = array();
+		foreach ( reset($args) as $key => $subfield ) {
```
Comment (ziodave) [inline src/admin/WL_Metabox/WL_Metabox_Field_address.php]: @pieroit do we need a `reset` here?

File: src/admin/WL_Metabox/WL_Metabox_Field_address.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	public function html() {
+
+		// Open main <div> for the Field
+		$html = "<div class='wl-field'>";
+
+		// Label
+		$html .= "<h3>$this->label</h3>";
+
+		// print nonce
+		$html .= $this->html_nonce();
```
Comment (ziodave) [inline src/admin/WL_Metabox/WL_Metabox_Field_address.php]: @pieroit you can squeeze the 3 above together.

File: src/admin/WL_Metabox/WL_Metabox_Field_address.php (hunk 1 above)
Comment (pieroit) [inline src/admin/WL_Metabox/WL_Metabox_Field_address.php]: I use there `reset` to take the first element of the array which in turn contains an array (the looped one).

Comment (ziodave) [inline src/admin/WL_Metabox/WL_Metabox_Field_address.php]: @pieroit so isn't this better?

  `$args[0]`

Comment (pieroit) [inline src/admin/WL_Metabox/WL_Metabox_Field_address.php]: It's an hash... something like:
```
array(
    'address' => array( ... looped elements ... )
);
```

Comment (ziodave) [inline src/admin/WL_Metabox/WL_Metabox_Field_address.php]: @pieroit ah okey, I understand. can u add this as a comment to the code :-)

**Pull request merged**